src/app/pages/system/enclosure/components/enclosure-side/enclosure-svg/enclosure-svg.component.ts

```diff
@@ -36,7 +48,7 @@ export class EnclosureSvgComponent implements OnDestroy {
   private keyDownListener: () => void;
   private clickListener: () => void;
 
-  protected svg = signal<SafeHtml | undefined>(undefined);
+  svg = signal<SafeHtml | undefined>(undefined);
```

Comment:
Marking something as public because it does not work in tests is a good indication that you shouldn't interact with it in the test.

Do this instead:
```ts
const svgCache = spectator.inject(SvgCacheService);
jest.spyOn(svgCache, 'loadSvg').mockReturnValue(of()); // Simulate long network request.
spectator.setInput('svgUrl', '/assets/m2.svg');
...
```